main.go

```diff
@@ -72,12 +84,15 @@ func apiEPGHandler(response http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(response, string(marshEPG))
 }
 
-func testHadndler(response http.ResponseWriter, r *http.Request) {
-	testHtml, err := ioutil.ReadFile("html/test.html")
-	if err != nil {
-		fmt.Println("ERROR:", err)
+func handleAPIError(response http.ResponseWriter, apiErrMassage, jsonErrMassage string) bool {
+	fmt.Println(apiErrMassage)
+	marshErrAPI, jsonErr := json.Marshal(apiResponse{apiErrMassage, nil})
+	if jsonErr != nil {
+		fmt.Printf(apiErrMassage, jsonErr)
```

Comment:
da brauchst eigl nur den jsonErr ausgeben, die apiErrMessage hast eh schon in zeile 88 ausgegeben